src/utils/player.ts

```diff
@@ -162,8 +162,9 @@ class Player {
       } else return null;
     }
     // 返回歌曲地址
-    // 客户端直接返回，网页端转 https
-    const url = isElectron ? songData.url : songData.url.replace(/^http:/, "https:");
+    // 客户端直接返回，网页端转 https, 并转换url以便解决音乐链接cors问题
+    const url = isElectron ? songData.url : songData.url.replace(/^http:/, "https:").replace(/m804\.music\.126\.net/g, 'm801.music.126.net').replace(/m704\.music\.126\.net/g, 'm701.music.126.net');
```

Comment:
直接通过字符串替换来解决CORS问题虽然能快速生效，但不够灵活和稳定。建议使用更通用的代理方式或配置CORS策略，以提高代码的健壮性和可维护性。

```suggestion
// 客户端直接返回，网页端转 https, 并通过代理解决音乐链接cors问题
    const url = isElectron 
      ? songData.url 
      : songData.url
          .replace(/^http:/, "https:")
          .replace(/m\d+\.music\.126\.net/g, 'proxy.music.126.net'); // 使用更通用的正则表达式和代理域名